api/express-gen-ts/spec/Users.spec.ts

```diff
@@ -0,0 +1,204 @@
+import app from '@server';
+import supertest from 'supertest';
+
+import { BAD_REQUEST, CREATED, OK } from 'http-status-codes';
+import { Response, SuperTest, Test } from 'supertest';
+import { IUser, User } from '@entities';
+import { UserDao } from '@daos';
+import { pErr, paramMissingError } from '@shared';
+
+describe('Users Routes', () => {
+
+    const usersPath = '/api/users';
+    const getUsersPath = `${usersPath}/all`;
+    const addUsersPath = `${usersPath}/add`;
+    const updateUserPath = `${usersPath}/update`;
+    const deleteUserPath = `${usersPath}/delete/:id`;
+
+    let agent: SuperTest<Test>;
+
+    beforeAll((done) => {
+        agent = supertest.agent(app);
+        done();
+    });
+
+    describe(`"GET:${getUsersPath}"`, () => {
```

Comment:
Is this stuff all auto-generated? I didn't think we were going to have users for this app - weren't we going to do magic links for everything?